pkg/apis/kubeone/helpers_test.go

```diff
@@ -75,83 +75,6 @@ func TestFeatureGatesString(t *testing.T) {
 	}
 }
 
-func TestContainerRuntimeConfig_MachineControllerFlags(t *testing.T) {
```

Comment:
This is also testing containerd, why are we removing this test?

Reply:
"Collateral damage". Probably I got too excited to remove docker and took some extra :shrug: 

Reverted back to original.